src/scrapers/mizrahi.ts

```diff
@@ -260,6 +276,24 @@ class MizrahiScraper extends BaseScraperWithBrowser<ScraperSpecificCredentials>
       balance: +response.body.fields?.Yitra,
     };
   }
+
+  private shouldMarkAsPending(txn: Transaction): boolean {
+    if (this.options.optInFeatures?.includes('mizrahi:pendingIfNoIdentifier') && !txn.identifier) {
+      debug(`Marking transaction '${txn.description}' as pending due to no identifier.`);
+      return true;
+    }
+
+    const genericDescriptions = ['העברת יומן לבנק זר מסניף זר'];
```

Comment:
Put this in the head of the file, I think it is good to hold all the "scraping keys" in one place